scenarios/command/c/command_client/main.c

```diff
@@ -0,0 +1,205 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <uuid/uuid.h>
+
+#include "mosquitto.h"
+#include "mqtt_callbacks.h"
+#include "mqtt_protocol.h"
+#include "mqtt_setup.h"
+
+#define PAYLOAD "\n\v\b\261Í\244\006\020\364\254\265d\022\nmobile-app"
+#define RESPONSE_TOPIC "vehicles/vehicle03/command/unlock/response"
+#define PUB_TOPIC "vehicles/vehicle03/command/unlock/request"
+#define COMMAND_CONTENT_TYPE "application/protobuf"
+#define COMMAND_TIMEOUT_SEC 10
+
+#define QOS 1
+#define MQTT_VERSION MQTT_PROTOCOL_V5
+
+#define UUID_LENGTH 16
+
+#define CONTINUE_IF_ERROR(rc)                                   \
+  do                                                            \
+  {                                                             \
+    if (rc != MOSQ_ERR_SUCCESS)                                 \
+    {                                                           \
+      printf("Error publishing: %s\n", mosquitto_strerror(rc)); \
+      mosquitto_property_free_all(&proplist);                   \
+      continue;                                                 \
```

Comment:
This should go back to the top of the do loop.

Reply:
Fixed with if(true)